share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
+    set -l _matched (string match -r "^ +([^ ]*) *([^ ]?.*)" $task)
+
+    if test -n "$_matched"
+        set -l _task_name $_matched[2]
```

Comment:
You know... 13 hours ago I would have told you that this could lead to an "Array index out of bounds" error. But then we merged #4151.

😺